core/backend/src/IpcHost.ts

```diff
@@ -224,6 +224,15 @@ class IpcAppHandler extends IpcHandler implements IpcAppFunctions {
   public async openStandalone(filePath: string, openMode: OpenMode, opts?: StandaloneOpenOptions): Promise<IModelConnectionProps> {
     return StandaloneDb.openFile(filePath, openMode, opts).getConnectionProps();
   }
+  public async openSnapshot(filePath: string, opts?: SnapshotOpenOptions | undefined): Promise<IModelConnectionProps> {
+    let resolvedFileName: string | undefined = filePath;
+    if (IModelHost.snapshotFileNameResolver) {
```

Comment:
i dont think anyone is actually using this fileNameResolver, id like to try and not carry fwd junk with us

Reply:
`IModelHost.snapshotFileNameResolver` is public and, if we start ignoring it, it would be a breaking change.

We could deprecate it now and stop using it in 5.0.